pkg/client/results/testdata/mockResults/plugins/ds-junit-01/ds-junit-01.golden.json

```diff
@@ -0,0 +1 @@
+{"name":"ds-junit-01","status":"passed","meta":null,"file":"","items":[{"name":"global","status":"passed","meta":null,"file":"","items":[{"name":"output.xml","status":"passed","meta":{"file":"results/global/output.xml"},"file":"","items":[{"name":"[k8s.io] Pods should be submitted and removed [NodeConformance] [Conformance]","status":"passed","meta":null,"file":"","items":null},{"name":"[sig-node] ConfigMap should fail to create ConfigMap with empty key [Conformance]","status":"passed","meta":null,"file":"","items":null},{"name":"[sig-storage] Downward API volume should set DefaultMode on files [LinuxOnly] [NodeConformance] [Conformance]","status":"passed","meta":null,"file":"","items":null},{"name":"[sig-storage] In-tree Volumes [Driver: local][LocalVolumeType: dir-link-bindmounted] [Testpattern: Dynamic PV (default fs)] subPath should support existing directories when readOnly specified in the volumeSource","status":"skipped","meta":null,"file":"","items":null},{"name":"[sig-storage] In-tree Volumes [Driver: rbd][Feature:Volumes] [Testpattern: Pre-provisioned PV (default fs)] subPath should support restarting containers using file as subpath [Slow]","status":"skipped","meta":null,"file":"","items":null}]}]}]}
```

Comment:
It looks like `null` fields are still being included in the output. I thought they would have been skipped with the `omitempty` tag.

Reply:
I can fix those up, though it doesnt matter. In this situation, the item is same once loaded/compared.

Reply:
Ah, sorry. I had a couple of comments which were outdated, one of which applied to the example test which _does_ show the empty fields being omitted. I thought these test files had been regenerated following those `omitempty` changes. Never mind this comment then!